odoo/osv/query.py

```diff
@@ -172,3 +172,6 @@ def add_joins_for_table(lhs):
 
     def __str__(self):
         return '<osv.Query: "SELECT ... FROM %s WHERE %s" with params: %r>' % self.get_sql()
+
+    def is_false(self):
+        return 'FALSE' in self.where_clause
```

Comment:
What if the where clause is this?
```sql
TRUE OR FALSE
```

Reply:
`where_clause` list is joined with `AND`: https://github.com/odoo/odoo/blob/95cc9791d6c59cc1c1545b9b39db246042362915/odoo/osv/query.py#L171